fplus-http-server/src/router/application.rs

```diff
@@ -244,11 +244,11 @@ pub async fn merged(query: web::Query<GithubQueryParams>) -> actix_web::Result<i
     }
 }
 
-#[post("/application/refill")]
-pub async fn refill(data: web::Json<RefillInfo>) -> actix_web::Result<impl Responder> {
-    match LDNApplication::refill(data.into_inner()).await {
-        Ok(applications) => Ok(HttpResponse::Ok().json(applications)),
-        Err(e) => Ok(HttpResponse::BadRequest().body(e.to_string())),
+#[post("/application/notify_refill")]
+pub async fn remind_refill(info: web::Json<NotifyRefillInfo>) -> impl Responder {
```

Comment:
```suggestion
pub async fn notify_refill(info: web::Json<NotifyRefillInfo>) -> impl Responder {
```
Endpoint is `/application/notify_refill`, data struct is `NotifyRefillInfo`, so it would make sense to call handler `notify_refill` as well?